packages/chaire-lib-backend/src/utils/processManagers/TrRoutingProcessManager.ts

```diff
@@ -14,6 +14,13 @@ import config from '../../config/server.config';
 // Set 2 threads as default, just in case we have more than one request being handled
 const DEFAULT_THREAD_COUNT = 2;
 
+type StartParameters = {
```

Comment:
Please add a command to mention what is that type and where it will be used.

Reply:
If it's of general use it should be in the ProcessManager file. Otherwise it whould be named TrRouting something...

Reply:
Done, this is for the TrRotuingProcessManager only